src/actions/projects/shareProject.js

```diff
@@ -9,31 +8,18 @@ import { AWS_STAGING_BUCKET } from '../../constants/aws';
 import s3 from '../../utils/s3';
 import store from '../../store';
 
-export default async function shareProject(id) {
-  const {
-    project: projectName,
-  } = ids.generate(id);
+export default async function shareProject(projectId) {
+  const shareKey = getProjectSharePath(projectId);
 
-  const shareKey = getProjectSharePath(id);
-
-  // TODO: I think we have to do something with this
-  // const shareUrl = getProjectSharePath(id, { asUrl: true });
   const illustrationIds = await store.illustrations.get();
   const illustrations = illustrationIds
-    .filter(([, data]) => data.project === projectName)
-    .map(([entryId, data]) => ({
-      id: entryId,
-      name: projectName,
-      slug: data.slug,
-    }));
+    .filter(([, data]) => data.project === projectId)
+    .map(([, data]) => data.slug);
```

Comment:
We have a util for this:
https://github.com/The-Politico/artisan/blob/v2.0/src/utils/store/getIllosInProject.js

Reply:
Also slug is no longer in the store for data:
https://github.com/The-Politico/artisan/blob/%40sprint/briz/fareed/src/constants/types.js#L23-L32

What you want to do is use ids.parse to get the illustration part of the ID.